polaris/polaris/sep24/tzinfo.py

```diff
@@ -0,0 +1,37 @@
+import pytz
+from datetime import datetime, timedelta, timezone
+
+from rest_framework.decorators import api_view, parser_classes, renderer_classes
+from rest_framework.parsers import JSONParser
+from rest_framework.renderers import JSONRenderer
+from rest_framework.request import Request
+from rest_framework.response import Response
+from django.contrib.sessions.backends.db import SessionStore
+
+from polaris.utils import render_error_response, getLogger
+
+
+logger = getLogger(__name__)
+
+
+@api_view(["POST"])
+@parser_classes([JSONParser])
+@renderer_classes([JSONRenderer])
+def post_tzinfo(request: Request) -> Response:
+    if not (
+        request.data.get("sessionId") and request.data.get("sessionOffset") is not None
+    ):
+        return render_error_response("missing required parameters")
+    now = datetime.now(timezone.utc)
+    offset = timedelta(minutes=request.data["sessionOffset"])
```

Comment:
`request.data["sessionOffset"]` is inconsistent with line:22:`request.data.get("sessionOffset")`

Reply:
Thats because the first time we check if the `sessionOffset` variable is present in `request.data`. If it didn't, then `request.data["sessionOffset"]`would throw an exception, whereas `.get("sessionOffset")` would return `None`.